_candidates_en/IppeiTakahashi_1.html

```diff
@@ -0,0 +1,19 @@
+---
+name: Ippei Takahashi
+title: Functional server side development by Play, Slick, Scalaz
+length: 40
+audience: Intermediate
+language: Japanese
+twitter: 
+github: ippei-takahashi
+icon: http://gravatar.com/avatar/e49eefa4110fda959f19da1341871b53
+organization: Recruit Marketing Partners Co., Ltd.
+---
+<p>
+In our team, we use Play framework, Slick, and Scalaz for new product's server side implementation.
+Slick after version 3.0 returns DB operation result by Future. And it can use Either for error handling. They are convenient, but they could be negative factor for code's readability. Monad and Monad transformer provided by Scalaz are useful for data operation wrapped by context(Future, Either etc..), and they aid high readable code what uses "for" operator.
```

Comment:
原文: バージョン3.0以降のSlickは、DB操作の結果をFutureに包んで返します。また、エラー処理にはEitherが使えます。これらは便利ですが、コードの可読性を下げる原因にもなります。Scalazが提供するモナドやモナド変換子は、FutureやEither等のコンテキストに包まれたデータの操作に有用で、for式による可読性の高いコードの作成を助けます。
訳案: Since Slick 3.0, it produces the result of DB operation encapsulated in Future, and you may handle errors using Either. These features are convenient, while make the code less readable. Scalaz provides Monad and Monad transformer that are useful for operating data wrapped in the context such as Future and Either, and help us to make our code more readable through "for" expression.

コンテキストが包んでいるのは、ここでは「操作」ではなく「データ」の方かと思います。